exercises/nodejs/src/game.js

```diff
@@ -1,172 +1,208 @@
-import generator from 'random-seed'
+import generator from "random-seed";
 
 var Game = function () {
-  var players = new Array()
-  var places = new Array(6)
-  var purses = new Array(6)
-  var inPenaltyBox = new Array(6)
+  var players = new Array();
+  var boardSquares = new Array(6);
```

Comment:
this could be a confusing name (although the fact that they are all arrays doesn't help). I would read this as there are only 6 board squares on the board. Whereas there are 6 positions in an array for each of the players position on the board